tests/contest/contest/src/tests/process_user/process_user.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::{Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, Spec, SpecBuilder, UserBuilder};
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec() -> Result<Spec> {
+    let user = UserBuilder::default()
+        .uid(10u32)
+        .gid(10u32)
+        .additional_gids(vec![5u32])
+        .umask(0o02u32)
+        .build()?;
+
+    let spec = SpecBuilder::default()
+        .process(
+            ProcessBuilder::default()
+                .user(user)
+                .build()
+                .expect("error in creating process config"),
```

Comment:
You need to set the process cmd to the runtimetest to actually run the internal test

Reply:
I add args to this commit.

https://github.com/youki-dev/youki/pull/2978/commits/ab34d99e2b64f595d18ca8086c3f12d6a13b53a2